tests/test_driver.py

```diff
@@ -151,6 +158,67 @@ def test_get(self, driver,
                        for tx in response if 'id' in tx['asset'])
 
 
+class TestOutputsEndpointMultiple:
```

Comment:
The pecularities of the outputs', blocks', and other entities' structures are decoupled from how we do requests so I do not see much help in repeating all these `TestOutputsEndpointMultiple`  and `TestBlocksEndpointMultiple` tests.

I would take a single simple case, e. g. this one:

```
block = driver_multiple_headers.blocks.retrieve(block_height=str(block_with_alice_transaction))
assert block
```

and try to assert it works for different node setups.

Reply:
Ok ill update accordingly.  but the idea behind is not to select some specific request case for testing multiple nodes running, i selected a random couple of endpoint requests which could represent the complete set and add the multiple nodes test case there, maybe we should add them to all of them?  on the other hand yeah i agree if the representative sample to test multiple nodes is the one you suggest ill remove current ones and add only ```blocks.retrieve```

Reply:
Yes, all endpoints use the same transport so I do not see why would we want to query every endpoint with the new setup.